language-design.md

```diff
@@ -37,8 +60,9 @@ you know a poorly designed language? What are the symptoms?
 How do the themes of _Growing a Language_ relate to the lab we did this week?
 
 **Response**
+In the lab we had everyone in the class working to improve the same 'language'. I think that that speaks to the bazaar way of building a language. -Daniel
 
-
+Agreed, although it was mostly the beginning stages of the bazaar style, since we didn't combine the best of all our work into a final language. -Aaron
 
```

Comment:
I agree, but it seems like there was more. What about the second half when we discussed how we each approached the problem, was their anything from that discussion that related?